docs/en/ingest-management/faq.asciidoc

```diff
@@ -219,6 +214,7 @@ token, which {kib} validates.
 [[how-are-secrets-secured]]
 == How are secrets secured when entered into integration policies or agent policies in {kib}?
 
+//This include is designed to break in 7.15 so we remember to verify that this info is still true
```

Comment:
Brilliant yet terrifying 🤣

Reply:
I like to live on the edge. :-) But seriously, we set ourselves up for cruft if we don't build in some guardrails, especially since reviews typically include diffs only.

Reply:
👍 🚢